test/unit/YawPitchControl/YawPitchControl.spec.js

```diff
@@ -128,7 +131,7 @@ describe("YawPitchControl", function() {
 			var callback = sinon.spy();
 
 			setTimeout(() => {
-				this.inst.on("change", callback);
+				this.inst.on("change", () => callback);
```

Comment:
Why did you make a change like this?

Reply:
@happyhj I changed this part to other forms because sometimes if fails this test on headless. But above code was not related. So I'll restore it.